### DISCUSSION TITLE
[Question]: Should I send these types of mistaken identifications to Frigate+?

### DISCUSSION THREAD

**Discussion by ElectronicBattle** [blakeblackshear/frigate#13028] [General Support] [answered]
### Describe the problem you are having

Clearly this specific identification is wrong. However, there *is* car to the right of the shot (behind the blue gate) which is frequently correctly detected:
![image](https://github.com/user-attachments/assets/73577f89-3ba0-418a-b764-c5f3d8624364)

I have had a handful like this, where the object detection bounding box doesn't coincide with the object. 

Another example here, but Frigate+ doesn't yet handle bird labels. Nevertheless this camera often sees birds and Frigate usually labels the birds correctly. This time, it hasn't and again, like the shot above, there's a bounding box  offset.
![image](https://github.com/user-attachments/assets/e52ddd32-f20d-45aa-b06f-40b2c8c56b5c)

I know there are some general guidelines, and I have been applying them (e.g. when a cat is mistaken for a dog, or when there are things in an image which have not been identified with the available labels at all) but I am interested in these specific cases.

- Ignore them? 
 
- Submit by agreeing that yes indeed, "this isn't a car" or "this isn't a bird" ?
 
- Submit as above but add corrections where possible i.e. draw the box in the correct place (when there are object labels available)?

 

### Version

0.14

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Frigate stats

```json
n/a
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

n/a

### Any other information that may be helpful

n/a

**Comment by NickM-27** [comment, marked as answer]
this almost seems like an incorrect configuration causing issues and not necessarily an error in the detection. I would suggest waiting to submit false positives to frigate+ until you are running a plus model.

**Reply by ElectronicBattle** [reply]
It wouldn't be the first time that my config was dubious!

I've submitted about 900 images from four cameras now. Probably 800 are true false positives ("no, it is not a dog. I will correct it to show a cat"), with another 150-200 being confirmatory ("yes, this *is* a dog - well done  ... but there is a car and a person in the background too, both of which I have marked up"). 

I have seen perhaps a dozen of these misaligned ones, so it hasn't proved burdensome.  

(I haven't submitted a model request yet because so far I have only used four out of a total of eight  cameras and I haven't even begun to look at the remaining  four).
